src/const_game.h

```diff
@@ -0,0 +1,45 @@
+// hunger thresholds and messages
```

Comment:
if these all came from game.cpp, I think it would be more productive to put them at the top of that file, otherwise discoverability is a bit lost.  I'm also not a fan of increasing the number of files, it makes the source even harder to browse.
If there are very many strings being treated like this, it would be good to define them in a cpp file and declare them as extern in a header file, otherwise a separate copy of the string will exist in each compilation unit that includes the header.